package.json

```diff
@@ -21,26 +25,38 @@
     "webhook": "./packages/cli/bin/n8n webhook",
     "worker": "./packages/cli/bin/n8n worker"
   },
+  "dependencies": {
+    "n8n": "*"
+  },
   "devDependencies": {
+    "@n8n_io/eslint-config": "*",
     "@types/jest": "^28.1.8",
+    "@types/node": "^16.11.22",
     "jest": "^28.1.3",
     "jest-environment-jsdom": "^28.1.3",
     "jest-mock": "^28.1.3",
-    "patch-package": "^6.4.7",
+    "nodemon": "^2.0.2",
+    "prettier": "^2.3.2",
     "rimraf": "^3.0.2",
     "run-script-os": "^1.0.7",
+    "supertest": "^6.2.2",
     "ts-jest": "^28.0.8",
-    "turbo": "1.2.15"
+    "ts-node": "^9.1.1",
```

Comment:
We only need `ts-node` for the `typeorm` command. In future I wonder if there's a better way to run migrations without it.

Reply:
we could compile the migrations before running them ?

Reply:
https://linear.app/n8n/issue/N8N-5312